react/features/toolbox/functions.web.js

```diff
@@ -4,6 +4,32 @@ import { hasAvailableDevices } from '../base/devices';
 
 declare var interfaceConfig: Object;
 
+/**
+ * Returns a set of button names to be displayed in the toolbox, based on the screen width.
+ *
+ * @param {number} width - The width of the screen.
+ * @returns {Set} The button set.
+ */
+export function getToolbarAdditionalButtons(width: number): Set<string> {
+    if (width <= 500) {
+        if (width <= 390) {
+            if (width <= 332) {
+                if (width <= 224) {
+                    return new Set();
```

Comment:
Not even 1 button?

Reply:
If this set is empty the `audio`, `video` & `hangup` will be displayed.  Also the rest of the buttons will show in the overflow menu.